network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -1428,4 +1535,162 @@ private <T extends IdentifiableAttributes> void deleteTemporaryLimits(UUID netwo
+            preparedStmt.setObject(1, networkUuid.toString());
+            preparedStmt.setInt(2, variantNum);
+            for (int i = 0; i < valuesForInClause.size(); i++) {
+                preparedStmt.setString(3 + i, valuesForInClause.get(i));
+            }
+
+            return innerGetReactiveCapabilityCurvePoints(preparedStmt);
+        } catch (SQLException e) {
+            throw new UncheckedSqlException(e);
+        }
+    }
+
+    public Map<OwnerInfo, List<ReactiveCapabilityCurvePointAttributes>> getReactiveCapabilityCurvePoints(UUID networkUuid, int variantNum, String columnNameForWhereClause, String valueForWhereClause) {
+        try (var connection = dataSource.getConnection()) {
+            var preparedStmt = connection.prepareStatement(QueryCatalog.buildReactiveCapabilityCurvePointQuery(columnNameForWhereClause));
+            preparedStmt.setObject(1, networkUuid.toString());
+            preparedStmt.setInt(2, variantNum);
+            preparedStmt.setString(3, valueForWhereClause);
+
+            return innerGetReactiveCapabilityCurvePoints(preparedStmt);
+        } catch (SQLException e) {
+            throw new UncheckedSqlException(e);
+        }
+    }
+
+    private Map<OwnerInfo, List<ReactiveCapabilityCurvePointAttributes>> innerGetReactiveCapabilityCurvePoints(PreparedStatement preparedStmt) throws SQLException {
+        try (ResultSet resultSet = preparedStmt.executeQuery()) {
+            Map<OwnerInfo, List<ReactiveCapabilityCurvePointAttributes>> map = new HashMap<>();
+            while (resultSet.next()) {
+
+                OwnerInfo owner = new OwnerInfo();
+                ReactiveCapabilityCurvePointAttributes reactiveCapabilityCurvePoint = new ReactiveCapabilityCurvePointAttributes();
+                // In order, from the QueryCatalog.buildReactiveCapabilityCurvePointQuery SQL query :
+                // equipmentId, equipmentType, networkUuid, variantNum, minQ, maxQ, p
+                owner.setEquipmentId(resultSet.getString(1));
+                owner.setEquipmentType(ResourceType.valueOf(resultSet.getString(2)));
+                owner.setNetworkUuid(UUID.fromString(resultSet.getString(3)));
+                owner.setVariantNum(resultSet.getInt(4));
+                reactiveCapabilityCurvePoint.setMinQ(resultSet.getInt(5));
+                reactiveCapabilityCurvePoint.setMaxQ(resultSet.getInt(6));
+                reactiveCapabilityCurvePoint.setP(resultSet.getInt(7));
+
+                List<ReactiveCapabilityCurvePointAttributes> reactiveCapabilityCurvePoints = map.get(owner);
+                if (reactiveCapabilityCurvePoints == null) {
+                    reactiveCapabilityCurvePoints = new ArrayList<>();
+                }
+                reactiveCapabilityCurvePoints.add(reactiveCapabilityCurvePoint);
+                map.put(owner, reactiveCapabilityCurvePoints);
```

Comment:
Then we overwrite the previous list ? hum this not good (I know it's my code :D )
We should fix this

Reply:
I fixed it in my PR for Tap changer steps ans temporaryLimits
Apply the code sample here :
```
 if (!map.containsKey(owner)) {
    List<ReactiveCapabilityCurvePointAttributes> reactiveCapabilityCurvePointsCollection = new ArrayList<>();
    reactiveCapabilityCurvePointsCollection.add(reactiveCapabilityCurvePoint);
    map.put(owner, reactiveCapabilityCurvePointsCollection);
} else {
    map.get(owner).add(reactiveCapabilityCurvePoint);
}
 ```

Reply:
Thanks, done